lib/config.js

```diff
@@ -44,5 +45,7 @@ module.exports = async (root) => {
 
   config.paths.push(root)
 
+  config.force = process.argv.indexOf('--force') !== -1
```

Comment:
total nit, but `process.argv.includes('--force')` is shorter and more clear (i think)